Curs/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,485 @@
+interviewApp.NewEvaluationPage = function(optionss = {}) {
```

Comment:
Any particular reason for the doubled `s` in `optionss`?

Reply:
Yes...I have an other variable called options in this file. I should rename it because options and optionss have different responsibilities.

Reply:
![ter](https://user-images.githubusercontent.com/31060543/30113701-47f0fee2-931e-11e7-9903-526c7d5ec03e.png)


It should better like that. :)